charts/vela-minimal/templates/defwithtemplate/suspend.yaml

```
@@ -11,6 +11,8 @@ spec:
   schematic:
     cue:
       template: |
-        // no parameters
-        parameter: {}
+        parameter: {
+        	// +usage=Specify the wait duration time to resume workflow such as "30s", "1min" or "2m15s"
+        	waitDuration: string
```

```suggestion
        	waitDuration ?: string
```

this must be an optional field.

---

Yes, i'll change it.